src/Evolution/VariableFixing/FixToAtmosphere.hpp

```diff
@@ -60,8 +76,29 @@ class FixToAtmosphere {
     static constexpr Options::String help = {
         "Density to impose atmosphere at. Must be >= rho_atm"};
   };
+  /// \brief For densities between DensityOfAtmosphere and
+  /// SmoothTransitionDensityCutoff the velocity is transitioned away from
+  /// atmosphere to avoid abrupt cutoffs.
+  struct TransitionDensityCutoff {
+    using type = double;
+    static type lower_bound() noexcept { return 0.0; }
+    static constexpr Options::String help = {
+        "For densities between DensityOfAtmosphere and "
+        "SmoothTransitionDensityCutoff the velocity is transitioned away from "
+        "atmosphere to avoid abrupt cutoffs."};
+  };
+  /// \brief The maximum magnitude of the velocity when the density is below
+  /// `TransitionDensityCutoff`
+  struct MaxVelocityMagnitude {
+    using type = double;
+    static type lower_bound() noexcept { return 0.0; }
+    static type upper_bound() noexcept { return 1.0; }
+    static constexpr Options::String help = {
+        "The maximum sqrt(v^i v^j gamma_{ij}) allowed."};
```

Comment:
I would state here also the clause "when the density is below TransitionDensityCutoff"